feature-libs/product-configurator/common/components/configurator-cart-entry-bundle-info/configurator-cart-entry-bundle-info.component.spec.ts

```diff
@@ -725,5 +726,248 @@ describe('ConfiguratorCartEntryBundleInfoComponent', () => {
         ).toBe(1);
       });
     });
+
+    describe('getButtonText', () => {
+      it("should return 'configurator.header.show'", () => {
+        component.hideItems = true;
+        fixture.detectChanges();
+        expect(
+          component.getButtonText().indexOf('configurator.header.show')
+        ).toBe(0);
+      });
+
+      it("should return 'configurator.header.hide'", () => {
+        component.hideItems = false;
+        fixture.detectChanges();
+        expect(
+          component.getButtonText().indexOf('configurator.header.hide')
+        ).toBe(0);
+      });
+    });
+
+    describe('getItemsMsg', () => {
+      it("should return 'configurator.a11y.cartEntryBundleInfoSg'", () => {
```

Comment:
Please state what this test is assuming (singular), otherwise one does not really understand the test description

Reply:
Done!